app/views/lab_results/_form.html.erb

```diff
@@ -0,0 +1,65 @@
+<section class="section">
+  <div class="container">
+    <%= form_for([patient, lab_result]) do |form| %>
+      <% if lab_result.errors.any? %>
+        <div class="notification is-danger is-light content">
+          <p class="subtitle">
+            <%= pluralize(lab_result.errors.count, "error") %> prohibited this lab result from being saved:
+          </p>
+          <ul>
+            <% lab_result.errors.each do |error| %>
+              <li><%= error.full_message %></li>
+            <% end %>
+          </ul>
+        </div>
+      <% end %>
+
+      <div class="field">
+        <%= form.label :effective, class: 'label' %>
```

Comment:
Does it make sense to change the title of this field from "Effective" to "Effective Date"? That way it could be consistent with how it is displayed in the Patient show page, and I think it makes it more clear to the user what this field is actually means.

Reply:
After discussion with Mick, best to leave as is